components/select_team/components/select_team_item.jsx

```diff
@@ -44,12 +44,13 @@ export default class SelectTeamItem extends React.PureComponent {
 
             showDescriptionTooltip = (
                 <OverlayTrigger
-                    trigger={['hover', 'focus', 'click']}
+                    trigger={['click']}
```

Comment:
Why is the tooltip's trigger changing? I could be wrong, but it doesn't look related to scrolling